packages/cli-dxns/src/handlers/seed.ts

```diff
@@ -85,11 +85,30 @@ export const seedRegistry = (params: Params) => async (argv: any) => {
     description: 'Base DXOS schema'
   };
 
+  const typeNameToCID: Record<string, CID> = {};
+
   for (const [typeName, fqn] of Object.entries(TYPES)) {
     verbose && log(`Registering ${typeName}..`);
 
     const cid = await client.registryClient.insertTypeRecord(root, fqn, meta);
     await client.registryClient.registerResource(domainKey, typeName, cid);
+    typeNameToCID[typeName] = cid;
+
+    if (typeName === 'type.service.ipfs') {
+      // Some fake data for ipfs service
+      const serviceData = {
+        type: 'ipfs',
+        kube: createCID().value,
+        extension: {
+          '@type': typeNameToCID['type.service.ipfs'],
+          protocol: 'ipfs/0.1.0',
+          addresses: [
+            '/ip4/123.123.123.123/tcp/5566'
+          ]
+        }
+      };
+      await client.registryClient.insertDataRecord(serviceData, typeNameToCID['type.service.ipfs']);
+    }
```

Comment:
Generally, `seed` command is used to pre-seed DXNS with data, required to operate, e.g. create dxos org, register types, etc. All dummy data should not be the part of seed;
I know you added it because there is no other way to test your changes; however, lets either:
- add TODO to remove it once we fully integrate everything, or
- add new command under dxns cli that would take care of creating dummy records for any tests we need to perform